client.go

```diff
@@ -480,6 +480,83 @@ func (c *Client) ListContainers() ([]string, error) {
 	return names, nil
 }
 
+func (c *Client) CopyImage(image string, dest *Client, name string, cp bool, aliases []string) error {
+	uri := c.url(shared.APIVersion, "images", image, "export")
+
+	req, err := http.NewRequest("GET", uri, nil)
```

Comment:
This is the third copy of this code we have now. Is there any way we can abstract it?